ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -430,84 +500,106 @@ class TreegraphSeries extends OrganizationSeries {
+                series.options.link && series.options.link.lineWidth
             );
-            return attrs;
+
+        if ((point as any).isLink) {
+            attribs.stroke = linkColor;
+            attribs['stroke-width'] = linkLineWidth;
+            delete attribs.fill;
+        } else {
+            if (borderRadius) {
+                attribs.r = borderRadius;
+            }
         }
-        return super.pointAttribs.apply(this, arguments);
+        return attribs;
     }
 
+    public drawPoints(): void {
+        super.drawPoints.apply(this, arguments);
+        const points = this.points;
+        this.points = this.links as any;
+        ColumnSeries.prototype.drawPoints.apply(this, arguments);
+        this.points = points;
+    }
     /**
      * Run translation operations for one node.
      * @private
      */
-    public translateNode(node: TreegraphNode): void {
+    public translateNode(point: TreegraphPoint): void {
         const chart = this.chart,
+            node = point.node,
             plotSizeY = chart.plotSizeY as number,
             plotSizeX = chart.plotSizeX as number,
             { ax, bx, ay, by } = this.layoutModifier,
             x = ax * node.xPosition + bx,
             y = ay * node.yPosition + by,
-            markerOptions = merge(this.options.marker, node.options.marker),
+            markerOptions = merge(this.options.marker, point.options.marker),
             symbol = markerOptions.symbol,
             height = node.nodeSizeY,
             width = node.nodeSizeX,
             reversed = this.options.reversed,
-            nodeX = chart.inverted ? plotSizeX - width / 2 - x : x - width / 2,
-            nodeY = !reversed ? plotSizeY - y - height / 2 : y - height / 2;
-
-        node.shapeType = 'path';
-        node.nodeX = node.plotX = nodeX;
-        node.nodeY = node.plotY = nodeY;
-        node.shapeArgs = {
+            nodeX = (node.x = chart.inverted ?
+                plotSizeX - width / 2 - x :
+                x - width / 2),
+            nodeY = (node.y = !reversed ?
+                plotSizeY - y - height / 2 :
+                y - height / 2);
+
+        point.shapeType = 'path';
+        point.plotX = nodeX;
+        point.plotY = nodeY;
+        point.shapeArgs = {
             d: symbols[symbol || 'circle'](nodeX, nodeY, width, height),
             x: nodeX,
             y: nodeY,
             width,
             height
         };
 
-        node.shapeArgs.display = node.hasShape() ? '' : 'none';
-        // Calculate data label options for the point.
-        node.dlOptions = TreegraphSeries.getDLOptions({
-            level: (this.mapOptionsToLevel as any)[node.level],
-            optionsPoint: node.options
-        });
         // Set the anchor position for tooltip.
-        node.tooltipPos = chart.inverted ?
-            [plotSizeY - y, plotSizeX - x] :
-            [x, y];
+        point.tooltipPos = chart.inverted ?
+            [plotSizeY - y, x] :
```

Comment:
It looks that something is not working correctly in here, maybe we should use `nodeX` and `nodeY`?